src/diffusers/loaders/single_file.py

```diff
@@ -49,6 +49,7 @@ def build_sub_model_components(
     model_type=None,
     image_size=None,
     torch_dtype=None,
+    is_playground_model=False,
```

Comment:
I don't think we need this argument if we can infer it from the checkpoint keys.

Reply:
> I don't think we need this argument if we can infer it from the checkpoint keys.

is it possible to push this fix to A1111? really need it, tks!